website/docs/r/virtual_machine.html.markdown

```diff
@@ -74,7 +74,7 @@ resource "opennebula_virtual_machine" "demo" {
 The following arguments are supported:
 
 * `name` - (Required) The name of the virtual machine.
-* `permissions` - (Optional) Permissions applied on virtual machine. Defaults to the UMASK in OpenNebula (in UNIX Format: owner-group-other => Use-Manage-Admin.
+* `permissions` - (Optional) Permissions applied on virtual machine. Defaults to the UMASK in OpenNebula (in UNIX Format: owner-group-other => Use-Manage-Admin).
```

Comment:
not related. 
there is already an issue to manage all Documentation issues (and there are a lot ;))

Reply:
Let me know if you need help with that, I've seen some formatting issues in the published docs. Which seems to be fine when I generated the website by running `make website`. I don't know if they are related with some kind of version problem.

(I'll remove this change from this PR)